benchmarks/bm_runner.py

```diff
@@ -22,13 +27,13 @@
 GH_REPORT_DIR = ROOT_DIR.joinpath(".github", "workflows", "benchmark_reports")
 
 # Common ASV arguments for all run_types except `custom`.
-ASV_HARNESS = "run {posargs} --attribute rounds=4 --interleave-rounds --show-stderr"
+ASV_HARNESS = "run {posargs} --attribute rounds=3 --interleave-rounds --show-stderr"
```

Comment:
I'm curious why `rounds` ended up changed here, I assume that this value is somewhat arbitrary and this at least keeps it cosistent with the rest of SciTools. I don't think this change is a problem, but I'd be interested if there was a motivation for it.

Reply:
`rounds=4` came from Iris originally. SciTools/iris#5949 and SciTools/iris#5981 made Iris' benchmarks accurate enough to reduce the number of rounds, without causing a load of false positive results (SciTools/iris#6234). So `rounds=3` is what ended up in the template.

I don't consider Iris-esmf-regrid to have any extra vulnerability to noise, so aligning with the template seemed sensible.